capstone-app/src/app/modules/history/wiki-results.service.spec.ts

```diff
@@ -50,8 +51,20 @@ describe('WikiResultsService', () => {
     req.flush(testResponse);
   });
 
+  it('searchNewLang(query) should return data', () => {
+    service.searchNewLang('Stillwater, (Oklahoma)', 'de', 'Geschichte').subscribe((result) => {
+      expect(result).toBeTruthy();
+    });
```

Comment:
Can we have this be more specific? Right now it's passing if any data at all is returned. Would it make more sense to make sure the response is sensible?

Reply:
Not really, it's doing very literally the exact same thing as the main search function (which is already tested) except using your wikiSearchHandler. I just want to make sure it makes a valid API call. The formatting of the string is already being tested.

Reply:
Since only fake data would be returned, adding a more specific test wouldn't accomplish much in my opinion.

Reply:
There's a lot of logic in the searchNewLang function and there's no test to see if the logic behaves correctly. Couldn't we use this test with this fake data to make sure that the fake data is parsed how we would expect?

Reply:
What I mean is that the functions that are called to parse that data are the exact same ones that are already being testes to ensure they work. We could add another test thought, I just wanted to explain why I didn't think it was necessary.